nbsmoke/lint/_pyflakes.py

```diff
@@ -0,0 +1,106 @@
+"""
+Copied out of pyflakes to allow additions (denoted by "nbsmoke
+addition"). Additions:
+
+ * return more info for use by caller
+
+ * support "noqa" in ipynb (of questionable value...)
```

Comment:
maybe add a note about hvplot?

Reply:
noqa support and this note predates the current PR, and was originally added (a couple of years ago now?) for a project that had a module imported for its side effects (unrelated to hvplot).

I've updated and clarified.

(What's added in this PR is support for ignoring flakes by regex, without requiring noqa at the flake itself.)